### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by DecliningLotus** [deepkit/deepkit-framework#145]
Similar to how [Next.js](https://github.com/vercel/next.js/tree/canary/examples) does it, I think having a directory of example projects would hugely benefit on-boarding new developers with this framework as well as giving experienced developers an easier overview of how all the code looks together as a final product. It could also serve well as starter templates.

Possible examples could showcase MongoDB, MySQL, RPC, caching, etc.

**Comment by marcj** [comment]
Absolutely, its planned to do that  + a videos series, after we are feature complete.

**Comment by DecliningLotus** [comment]
Out of curiosity, what's the roadmap/when do you expect to be feature-complete? Or that's still TBD?

**Comment by marcj** [comment]
There is no public roadmap, but I've put together a small (always changing) list of things that is being worked on: https://github.com/deepkit/deepkit-framework/projects/1. That should give some insights on what to expect before we consider it feature-complete and are investing time and money into marketing material.

**Comment by DecliningLotus** [comment]
That looks great! I can't wait!
